<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>314</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Beat Analysis</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>280</y>
      <width>451</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushOpen">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>279</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushAnalyse">
    <property name="geometry">
     <rect>
      <x>502</x>
      <y>276</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Analyse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushPlay">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>275</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>651</width>
      <height>261</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>649</width>
       <height>259</height>
      </rect>
     </property>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>651</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="lblBpm">
    <property name="geometry">
     <rect>
      <x>665</x>
      <y>220</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>BPM</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
